### PR TITLE
Added the usage of tilde (~) at the beginning of a path

### PR DESCRIPTION
**Pull request by Livila**

### CODE REVIEW

File: src/main/java/CodeCheck/Main.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 public class Main {
     public static void main(String[] args) throws Exception {
         Log.log("Starting up...");
-        Log.log("Running on %s.", "\n");
+        Log.log("Running on %s."); //Log.log("Running on %s.%n", System.getProperty("os.name"));
```
Comment (FeKozma) [inline src/main/java/CodeCheck/Main.java]: revert?

File: src/test/java/CodeCheckTest.java [hunk 2]
```diff
@@ -17,16 +17,19 @@ public class CodeCheckTest {
...
     private final String resultFileName = "test-result_0.txt";
 
     @Test
     public void executeTest() throws Exception {
-        delete(FileSystems.getDefault()
-                .getPath("").toAbsolutePath() + File.separator + resultFolder);
+        delete(FileSystems.getDefault().getPath("").toAbsolutePath() + File.separator + resultFolder);
 
-        Util.createFile(testConfPath, true);
+        Log.debug("#--- First: " + FileSystems.getDefault().getPath("").toAbsolutePath() + File.separator + resultFolder);
```
Comment (FeKozma) [inline src/test/java/CodeCheckTest.java]: only one would suffice

File: src/main/java/CodeCheck/Main.java (hunk 1 above)
Comment (FeKozma) [reply src/main/java/CodeCheck/Main.java]: tror du gjorde det

**Comment by FeKozma** [comment]
Looks good to me (y)

File: src/main/java/CodeCheck/Main.java (hunk 1 above)
Comment (Livila) [reply src/main/java/CodeCheck/Main.java]: Yes, it was removed in a later commit. :)

File: src/test/java/CodeCheckTest.java (hunk 2 above)
Comment (Livila) [reply src/test/java/CodeCheckTest.java]: This particular case was fixed in another commit. But I get what you mean.

**Pull request merged**